package.json

```diff
@@ -32,5 +32,13 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
```

Comment:
Opps, you need to add `prop-types`

screenshot from pathwright;
![image](https://user-images.githubusercontent.com/31897434/83758107-6a18ab80-a669-11ea-85aa-1e6c5d8e5f0c.png)